### DISCUSSION TITLE
Custom attribute == Additional attributes?

### DISCUSSION THREAD

**Discussion by bs27975** [keepassxreboot/keepassxc#10428] [Q&A] [answered]
https://keepassxc.org/docs/KeePassXC_UserGuide references `O – Custom Attribute (SEARCH_IN only)` in _Entry Cross-Reference_.

Are _Custom Attribute_ s (indirectly referenced via `{REF:𝑥@O:}`), _Additional attributes_ (auto-typed via `{S:}`)?

If so, perhaps the doc could merge references to the two (as _Additional custom attributes_ ?), so searching on _custom attribute_ within the doc finds the _additional attribute_ description, as well?

**Comment by droidmonkey** [comment, marked as answer]
Yes those are the same thing

**Reply by bs27975** [reply]
Thanks for the confirm. Ran into this when searching 'custom' in the docs, and getting no hits.
